### ISSUE TITLE
Sidebars from WordPress?

### ISSUE THREAD

**Issue by joshsmith01** [ivandoric/Gatsby-WordPress#1]
Is there a way that you've found to pull in sidebar data into Gatsby? I've installed a [WP REST API Sidebars](https://github.com/martin-pettersson/wp-rest-api-sidebars) but it doesn't pull in any of the widget's data. I also tried to pull in data with a custom GraphyQL query, but that didn't help.

**Comment by ivandoric** [comment]
No, sorry. Haven't played with at all. I think the gatsby-source-wordpress plugin should be extended to handle sidebars. I'm a noob at  React and Gatsby so can't give you any helpful tips, sorry.

**Comment by ph00lt0** [comment]
This issue should be submitted to the Gatsby repo i.e. like [here](https://github.com/gatsbyjs/gatsby/issues/7465)
